test/controllers/navigation/NavigatorTest.scala

```diff
@@ -20,28 +21,62 @@ import scala.concurrent.duration.FiniteDuration
 
 class NavigatorTest extends WordSpec with Matchers with MockitoSugar with ExportsDeclarationBuilder {
 
+  private val call = Call("GET", "url")
   private val config = mock[AppConfig]
   private val navigator = new Navigator(config)
 
-  "Go to Draft Confirmation" should {
+  "Continue To" should {
     val updatedDate = LocalDate.of(2020, 1, 1)
     val expiryDate = LocalDate.of(2020, 1, 1).plusDays(10)
 
     val declaration = aDeclaration(withUpdateDate(updatedDate))
-    val authenticatedRequest = AuthenticatedRequest(FakeRequest("GET", "uri")
-      .withSession(ExportsSessionKeys.declarationId -> "declarationId"), mock[SignedInUser])
-    val request = JourneyRequest(authenticatedRequest, declaration)
+    def authenticatedRequest(action: Option[FormAction]) = AuthenticatedRequest[AnyContent](
+      FakeRequest("GET", "uri")
```

Comment:
there are utility methods in AuthMocks to create fake AuthenticateRequest

Reply:
This class depends on the structure of the request so I would prefer to know exactly what is in it.

A change to the AuthMocks could mean this test passes/fails unintentionally